styles/hydrogen.less

```diff
@@ -5,488 +5,275 @@
+        display: inline;
     }
+}
 
-    .bubble-result-container {
-        width: fit-content !important;
-        height: fit-content !important;
-        max-width: unset !important;
-        max-height: unset !important;
-        overflow: hidden !important;
-        white-space: normal;
-        code {
-            font-size: @font-size;
-            font-family: inherit;
-
-            color: lighten(@syntax-text-color, 10%);
-            background-color: @bubble-background;
-
-            margin: 0;
-            padding: 0;
-            border: none;
-
-            white-space: pre;
-        }
-
-        img {
-            background-color: white;
-        }
+.nteract-display-area-stderr {
+  color: @hydrogen-error-color;
+}
 
-        svg:first-child {
-          background-color: white;
-        }
+// ----------- Inline Output --------------
+.inline-container {
+  color: @hydrogen-inline-color;
+  background: @hydrogen-inline-background;
+  border-radius: @component-border-radius;
+  box-shadow: 0 0 3px 0 rgba(0,0,0,0.3);
+}
 
-        video, img, svg {
-            display: block;
-        }
+.inline-container .cell_display {
+  padding: 0 0.25em 0 0.25em;
+}
 
-        td, th {
-            border: 1px solid darken(@bubble-background, 10%);
-            padding: 0.35em 0.7em;
-            font-size: @font-size;
-        }
+.inline-container.icon::before {
+  font-size: 80%;
+  width: 1.2em;
+  margin: 0;
+}
 
-        .MathJax_SVG_Display {
-            margin: 0;
-        }
+// ---------- Multiline Output ------------
 
-        .MathJax_SVG {
-            .noError {
-                padding: 0;
-                border: none;
-                color: @error-base-color
-            }
-            svg {
-                background-color: unset;
-                margin: 0 !important;
-                display: inline;
-            }
-        }
+.multiline-container {
+  display: flex;
+  border: solid 1px @base-border-color;
+}
 
-        .modebar-group {
-          display: flex;
-        }
-    }
+.multiline-container .cell_display {
+  padding: @component-padding;
+}
 
-    .bubble-error-container {
-        &.plain-error {
-            code {
-                color: @error-base-color;
-            }
-        }
+.multiline-container .toolbar {
+  border-left: solid 1px @base-border-color;
+  display: flex;
+  flex-direction: column;
+}
 
-        code {
-            font-size: @font-size;
-            font-family: inherit;
+.multiline-container .toolbar > :not(:first-child) {
+  border-top: solid 1px @base-border-color;
+}
 
-            color: lighten(@syntax-text-color, 10%);
-            background-color: @bubble-background;
+.multiline-container .toolbar .icon::before {
+  margin: 0;
+  width: 2.5em;
+  line-height: 2.5em;
+  height: 2.5em;
+  cursor: pointer;
+}
 
-            margin: 0;
-            padding: 0;
-        }
-    }
+.multiline-container .toolbar .icon:hover {
+  color: @hydrogen-inline-color;
```

Comment:
I like to have icons highlight when hovering personally.  I think toolbar icons should include `background-color: @background-color-highlight;` (We may not even need the `color` property line above) 

Here is how that looks (icons centered as well in this example)
![background-icon](https://cloud.githubusercontent.com/assets/10860657/26086644/8dbfd806-39b1-11e7-9149-e4d891d0455b.gif)